templates/master.yaml.tmpl

```diff
@@ -1989,6 +1989,14 @@ storage:
       contents:
         inline: |
             #!/bin/bash
+            {{if eq .Provider "azure" }}
```

Comment:
i don't think this is realible way to check this

i see in kernel log that initially eth0 is eth0

```
WARNING: CONTROL-PLANE MASTER | master0 waagent # journalctl -k | grep eth0
Nov 21 10:36:47 localhost kernel: IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
Nov 21 10:36:47 localhost kernel: IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
Nov 21 10:38:13 master0 kernel: docker0: port 2(veth0ce66fa) entered blocking state
Nov 21 10:38:13 master0 kernel: docker0: port 2(veth0ce66fa) entered disabled state
Nov 21 10:38:13 master0 kernel: device veth0ce66fa entered promiscuous mode
Nov 21 10:38:13 master0 kernel: IPv6: ADDRCONF(NETDEV_UP): veth0ce66fa: link is not ready
Nov 21 10:38:13 master0 kernel: docker0: port 2(veth0ce66fa) entered blocking state
Nov 21 10:38:13 master0 kernel: docker0: port 2(veth0ce66fa) entered forwarding state
Nov 21 10:38:13 master0 kernel: docker0: port 2(veth0ce66fa) entered disabled state
Nov 21 10:38:13 master0 kernel: eth0: renamed from veth24d21b4
Nov 21 10:38:13 master0 kernel: eth0: renamed from veth16184a4
Nov 21 10:38:14 master0 kernel: IPv6: ADDRCONF(NETDEV_CHANGE): veth0ce66fa: link becomes ready
Nov 21 10:38:14 master0 kernel: docker0: port 2(veth0ce66fa) entered blocking state
Nov 21 10:38:14 master0 kernel: docker0: port 2(veth0ce66fa) entered forwarding state
Nov 21 10:38:14 master0 kernel: eth0: renamed from veth0e99e68
Nov 21 10:38:14 master0 kernel: veth0e99e68: renamed from eth0
Nov 21 10:38:14 master0 kernel: docker0: port 2(veth0ce66fa) entered disabled state
Nov 21 10:38:15 master0 kernel: veth16184a4: renamed from eth0
Nov 21 10:38:15 master0 kernel: veth24d21b4: renamed from eth0
Nov 21 10:38:15 master0 kernel: docker0: port 2(veth0ce66fa) entered disabled state
Nov 21 10:38:15 master0 kernel: device veth0ce66fa left promiscuous mode
Nov 21 10:38:15 master0 kernel: docker0: port 2(veth0ce66fa) entered disabled state

```